core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/appRoutesModels.scala

```diff
@@ -37,6 +38,7 @@ final case class CreateAppRequest(kubernetesRuntimeConfig: Option[KubernetesRunt
 
 final case class GetAppResponse(appName: AppName,
                                 cloudContext: CloudContext,
+                                region: RegionName,
```

Comment:
Remind me - this endpoint is for GCP apps only, right?

Reply:
(these endpoints, I should say; get and list)

Reply:
Hmm, I thought it was for Azure too

Reply:
I'm not sure if Google's RegionName and Azure's region values are compatible 🤔

Reply:
Can Scala do union types?

Reply:
Or would we need to have two different valid signatures at the API doc level as well as the routes level?